client/api.js

```diff
@@ -0,0 +1,12 @@
+import request from 'superagent'
+
+const userUrl = 'http://localhost:3000/api/v1'
```

Comment:
I would use the variable name `apiUrl` as `userUrl` implies that it will include the `/user` route, which it does not.

Reply:
This is complete